src/main/app/src/flex.less

```diff
@@ -0,0 +1,267 @@
+.flex-row {
+  display: flex;
+  flex-direction: row;
+}
+
+.flex-col {
+  display: flex;
+  flex-direction: column;
+}
+
+.grid {
+  display: grid;
+}
+
+.flex-wrap {
+  flex-wrap: wrap;
+}
+
+.flex-col-xs {
+  @media screen and (max-width: 599px) {
+    flex-direction: column !important;
+  }
+}
+
+.flex-col-md {
+  @media screen and (max-width: 959px) {
+    flex-direction: column !important;
+  }
+}
+
+.flex-row-md {
+  @media screen and (max-width: 959px) {
+    flex-direction: row !important;
+  }
+}
+
+.flex-col-lg {
+  @media screen and (max-width: 1279px) {
+    flex-direction: column !important;
+  }
+}
+
+.flex-col-xl {
+  @media screen and (max-width: 1399px) {
+    flex-direction: column !important;
+  }
+}
+
+.flex-row-lg {
+  @media screen and (max-width: 1279px) {
+    flex-direction: row !important;
+  }
+}
+
+.gap-5 {
+  gap: 5px;
+}
+
+.gap-8 {
+  gap: 8px;
+}
+
+.gap-10 {
+  gap: 10px;
+}
+
+.gap-12 {
+  gap: 12px;
+}
+
+.gap-16 {
+  gap: 16px;
+}
+
+.gap-20 {
+  gap: 20px;
+}
+
+.space-between {
+  justify-content: space-between;
+}
+
+.justify-center {
+  justify-content: center;
+}
+
+.justify-content {
+  justify-content: flex-end;
+}
+
+.justify-start {
+  justify-content: flex-start;
+}
+
+.justify-end {
+  justify-content: flex-end;
+}
+
+.items-center {
+  align-items: center;
+  align-content: center;
+}
+
+.items-start {
+  align-items: start;
+  align-content: start
+}
+
+.items-end {
+  align-items: end;
+  align-content: end
+}
+
+.stretch {
+  align-items: stretch;
+  align-content: stretch;
+}
+
+.flex {
+  flex: 1 1 auto;
+}
+
+.flex-1 {
+  flex: 1
+}
+
+//flex: 1 1 0 (grow, shrink, basis 0). width: 100%;
+.flex-grow {
+  flex-grow: 1;
+}
+
+// flex: 1 1 auto (grow, shrink, basis auto).
+.flex-nogrow {
+  flex-grow: 0;
+}
+
+.hide {
+  display: none !important;
+}
+
+.hide-xs {
+  @media screen and (max-width: 599px) {
+    display: none !important;
+  }
+}
+
+.hide-gt-xs {
+  @media screen and (min-width: 600px) {
+    display: none !important;
+  }
+}
+
+
+.hide-sm {
+  @media screen and (max-width: 959px) {
+    display: none !important;
+  }
+}
+
+.hide-gt-sm {
+  @media screen and (min-width: 960px) {
+    display: none !important;
+  }
+}
+
+
+.w15 {
+  min-width: calc(15% - 20px);
+  flex-basis: calc(15% - 20px);
+}
+
+.w25 {
+  min-width: calc(25% - 20px);
+  flex-basis: calc(25% - 20px);
+}
+
+.w33 {
+  min-width: calc(33% - 20px);
+  flex-basis: calc(33% - 20px);
+}
+
+.w40 {
+  min-width: calc(40% - 20px);
+  flex-basis: calc(40% - 20px);
+}
+
+.w50 {
+  min-width: calc(50% - 20px);
+  flex-basis: calc(50% - 20px);
+}
+
+.w60 {
+  min-width: calc(60% - 20px);
+  flex-basis: calc(60% - 20px);
+}
+
+.w66 {
+  min-width: calc(66% - 20px);
+  flex-basis: calc(66% - 20px);
+}
+
+.w75 {
+  min-width: calc(75% - 20px);
+  flex-basis: calc(75% - 20px);
+}
+
+.w85 {
+  min-width: calc(85% - 20px);
+  flex-basis: calc(85% - 20px);
+}
+
+.w100 {
+  min-width: 100%;
```

Comment:
Mist hier -20px?

Reply:
nee, niet nodig. is voor wrapping (maximale gap), w100 zorgt altijd voor wrap.